### ISSUE TITLE
WCAG 2.2 本体 : 用語「慣用句 (idiom) 」の注記

### ISSUE THREAD

**Issue by caztcha** [waic/wcag22#2190]
原文
> Idioms cannot be translated directly, word for word, without losing their (cultural or language-dependent) meaning.	

現行訳（てにおはの修正）
> 慣用句は、その (文化的又は言語依存の) 意味を失わせることなく、直接に、逐語的に翻訳することはできない。

修正案
> 慣用句を、その (文化的又は言語依存の) 意味を失わせることなく、直接に、逐語的に翻訳することはできない。
